go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/grafana/dskit v0.0.0-20221216094413-a9826f9b0468
 	github.com/grafana/e2e v0.1.1-0.20221115065638-fe4609fcbc71
-	github.com/hashicorp/golang-lru v0.5.4
```

Comment:
There are many vendor updates here. Are all of these required?

Reply:
Unfortunately, it looks like it, or we would need to tweak it in the go.mod using some replace?
I suppose that was caused by [the upgrade of the golang-lru in alertmanager](https://github.com/prometheus/alertmanager/commit/c2b2defe48e7afe1db59c88da5643d0c920634d2) :/